scripts/compsets/exglobal/exglobal_fcst_nems.sh

```diff
@@ -1348,6 +1343,7 @@ if [[ $NEMS = .true. ]] ; then
   fi
 fi # NEMS
 
+module list
```

Comment:
maybe we don't want this

Reply:
it's just a more verbose output in the logs, might be helpful?

Reply:
yeah, it winds up being useful positioning for debugging in the event the module load fails -- don't know if there's an easier way to error check and exit prior to this point